README.md

```diff
@@ -94,6 +94,43 @@ First, install it: `npm install --save-dev aliasify`
 }
 ```
 
+## Usage with Rollup
+
+Using `preact-compat` with rollup requires [rollup-plugin-alias](http://npm.im/rollup-plugin-alias), along with a few other plugins
+
+First, install it: `npm install --save-dev rollup-plugin-alias rollup-plugin-buble rollup-plugin-commonjs rollup-plugin-node-resolve rollup-plugin-replace`
+
+... then in your `rollup.config.js`, configure the alias plugin to point to `react` and `react-dom`, along with the rest of the plugins:
+
+```js
+import alias from "rollup-plugin-alias";
+import buble from "rollup-plugin-buble";
+import resolve from "rollup-plugin-node-resolve";
+import replace from "rollup-plugin-replace";
+import commonjs from "rollup-plugin-commonjs";
+// ...
+
+export default {
+    // ...
+    plugins: [
+        // ...
+        resolve({
+            jsnext: true // preact
+        }),
+        alias({
```

Comment:
oh, that's a cool idea. Although the function shorthand may confuse users that want to use more than just the preact aliases, since they would need to be a little more verbose and use destructuring with the regular object syntax.